src/components/Tag/index.tsx

```diff
@@ -1,115 +1,103 @@
-import { Theme, css } from '@emotion/react'
 import styled from '@emotion/styled'
-import PropTypes from 'prop-types'
 import { MouseEventHandler, ReactNode } from 'react'
-import Box, { BoxProps } from '../Box'
-import Icon from '../Icon'
+import { Color } from '../../theme'
+import Icon, { IconName } from '../Icon'
 import Loader from '../Loader'
 import Touchable from '../Touchable'
+import Typography from '../Typography'
 
-const disabledStyles = css`
-  opacity: 0.5;
-`
-
-export const variantsContainer = {
-  base: ({ theme }: { theme: Theme }) => css`
-    background-color: ${theme.colors.neutral.backgroundStrong};
-    height: 24px;
-    padding-left: 8px;
-    padding-right: 8px;
-  `,
-  bordered: ({ theme }: { theme: Theme }) => css`
-    padding: 8px;
-    border: 1px solid ${theme.colors.neutral.borderWeak};
-  `,
-}
-
-type TagVariant = keyof typeof variantsContainer
-
-const variantStyles = ({
-  variant,
-  ...props
-}: {
-  variant: TagVariant
-  theme: Theme
-}) => variantsContainer[variant]?.(props)
+const StyledContainer = styled('span', {
```

Comment:
One last thing, is there a reason to put a `span` instead of a `div`? 

From what I see we were using a `div` before so this change might be breaking at some places 😛